src/contributing.md

```diff
@@ -96,6 +96,21 @@ For a full list of possible `groupname` check the `adhoc_groups` section at the
 or the list of teams in the [rust-lang teams
 database](https://github.com/rust-lang/team/tree/master/teams).
 
+---
+
+**Please notice**: we receive way more pull requests than we have reviewers available, many of them
```

Comment:
I don't like this wording. It makes it sounds like contributing isn't worthwhile since the PR won't be reviewed anyways.

Reply:
clearly not what I meant :-) Suggestions?

That sentence wants to set the expectations for contributors: be patient and use correctly the review flags. How about:

> Our pool of pull requests reviewers is often working at capacity, many of them are contributing on a volunteer basis. In order to minimize review delays [...]

Reply:
this sounds better 👍

Reply:
cool, thanks for proof-reading!